includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -534,8 +541,11 @@ private function get_courses( $args ) {
 			$course_args['s'] = $args['search'];
 		}
 
+		add_filter( 'posts_clauses', [ $this, 'add_days_to_completion_to_courses_queries' ] );
 		// Using WP_Query as get_posts() doesn't support 'found_posts'
-		$courses_query     = new WP_Query( apply_filters( 'sensei_analysis_overview_filter_courses', $course_args ) );
+		$courses_query = new WP_Query( apply_filters( 'sensei_analysis_overview_filter_courses', $course_args ) );
+		remove_filter( 'posts_clauses', [ $this, 'add_days_to_completion_to_courses_queries' ] );
```

Comment:
Just adding a note here that at some point this might have to be combined with the [lesson completion count filter](https://github.com/Automattic/sensei/pull/4873/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR558-R561). It's worth checking how this would work out.